lib/std/ascii.zig

```diff
@@ -272,15 +233,50 @@ pub fn toLower(c: u8) u8 {
 }
 
 test "ascii character classes" {
-    const testing = std.testing;
```

Comment:
What do you think of a test to compare table vs runtime values?

Reply:
Are you talking about testing both the naive implementations and the table implementations? This does not seem to be required because unless the tests are compiled with `ReleaseSmall` (which they will probably not), the table implementation will be used which ultimately depends on the naive implementations to generate the table, so basically both implementations are already tested by this.